### ISSUE TITLE
SuppressionCommentFilter: offComment should be included in the suppression (Eclipse formatter tags case)

### ISSUE THREAD

**Issue by stechio** [checkstyle/checkstyle#11504]
Check documentation: https://checkstyle.org/config_filters.html#SuppressionCommentFilter

```bash
$ javac Test.java

$ cat test-checkstyle.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
          "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name = "Checker">
  <module name="TreeWalker">
    <module name="CommentsIndentation">
      <property name="tokens" value="SINGLE_LINE_COMMENT, BLOCK_COMMENT_BEGIN"/>
    </module>
    <module name="SuppressionCommentFilter">
      <property name="offCommentFormat" value="@formatter:off"/>
      <property name="onCommentFormat" value="@formatter:on"/>
      <property name="checkFormat" value="CommentsIndentation"/>
    </module>
  </module>
</module>

$ cat Test.java
public class Test {
  public static void main(String[] args) {
// @formatter:off
    System.out.printf("A random %s call with several %s to fancifully format for stylecheck %s suppression testing...",
                  "method",
          "args",
  "rule"
      );
// @formatter:on
  }
}

$ java -jar checkstyle-10.1-all.jar -c test-checkstyle.xml Test.java
Starting audit...
[ERROR] Test.java:9:1: Comment has incorrect indentation level 0, expected is 4, indentation should be the same level as line 4. [CommentsIndentation]
Audit done.
Checkstyle ends with 1 errors.
```

---

EXPECTED BEHAVIOR
Using Eclipse's `@formatter` tags to disable formatting of certain code portions, I have the habit of keeping their respective comments at indentation level zero, much like C-preprocessor directives, to stick out of the actual code (instead of aligning to the current indentation level).

That would obviously trigger the `CommentsIndentation` rule, so I have to suppress it in those code portions.

`SuppressionCommentFilter` seems quite fitting for the purpose, as it maps just a couple of comment patterns as filter bounds; the PROBLEM here is that, whilst rule suppression _includes_ the opening comment (so-called offComment), it _excludes_ the closing comment (so-called onComment). Considering that _the Eclipse formatter honors BOTH tag comments (it doesn't format either)_, it would seem legit to expect a corresponding behavior from checkstyle (that is, the closing comment should be included in the suppression too).

**Comment by romani** [comment]
Looks like valid issue.

@rnveach , please confirm.
